web/libs/editor/src/tags/object/TimeSeries/ChannelLegend.module.scss

```
@@ -0,0 +1,31 @@
+.channelLegend {
+  --marker-size: 10px;
+  --marker-color:  var(--color-primary-surface, #4C5FA9);
+  --marker-hidden-color: var(--color-neutral-surface, #F9F8F6);
```

It's better to not default semantic tokens as we won't know up front if the correct dark mode compatible colours would be applying.

```suggestion
  --marker-color:  var(--color-primary-surface);
  --marker-hidden-color: var(--color-neutral-surface);
```

---

How can we have a dark mode without setting its own values?

---

The defaults here are not required and also can cause issues if there were to be styles that were not getting applied correctly, thus it is safer to not set defaults to CSS vars, since the defaults are not dark mode compatible themselves and would mask problems which we would otherwise visually see very simply in a review of the running feature. We have already run into that a bunch in the work done to make the application dark mode compatible and theme ready, which makes it hard to spot what in particular is wrong.